src/linker/Linker.Steps/ResolveFromXmlStep.cs

```diff
@@ -274,7 +274,6 @@ protected virtual void ProcessType (TypeDefinition type, XPathNavigator nav)
 
 			if (Annotations.IsMarked (type)) {
 				var duplicateLevel = preserve != TypePreserve.Nothing ? preserve : nav.HasChildren ? TypePreserve.Nothing : TypePreserve.All;
-				Context.LogWarning ($"Duplicate preserve of '{type.FullName}' in '{_xmlDocumentLocation}'", 2025, _xmlDocumentLocation);
```

Comment:
Is the `if` branch this is nested in useful for anything?

Reply:
Ops. It's not, we should remove it as well.